src/controllers/auth.ts

```diff
@@ -163,7 +163,7 @@ class AuthControls extends AuthServices {
                 return;
             }
 
-            const token = await this.tokenService.createToken({ id: Number(user.id), role: String(user.role) });
+            const token = await this.tokenService.createToken({ id: Number(user.id), role: String(user.role), email: String(user.email) });
```

Comment:
saved the user.email to the request, so cases where we want to send an email the sender email can be easily retrieved